src/components/graph/search/GraphSearchInput.tsx

```diff
@@ -281,15 +283,44 @@ function GraphSearchInput(props: Props) {
 		}
 	}
 
+	function preventDefaultHandlers(handlers: Object) {
+		const newHandlers: any = {};
```

Comment:
let's not use any
instead we can use
```
	type KeyboardHandler = {
		[key: string]: (keyEvent?: KeyboardEvent | undefined) => void;
	};
```

and make handlers type `KeyboardHandler`
and `const newHandlers: KeyboardHandler`

Reply:
I will fix it, thanks